openverse_catalog/dags/providers/provider_api_scripts/smk.py

```diff
@@ -1,223 +1,180 @@
 import logging
 
+from common import constants
 from common.licenses import get_license_info
 from common.loader import provider_details as prov
-from common.requester import DelayedRequester
-from common.storage.image import ImageStore
-from requests.exceptions import JSONDecodeError
+from providers.provider_api_scripts.provider_data_ingester import ProviderDataIngester
 
 
-logging.basicConfig(
-    format="%(asctime)s - %(name)s - %(levelname)s:  %(message)s", level=logging.INFO
-)
 logger = logging.getLogger(__name__)
 
-LIMIT = 2000
-DELAY = 5
-RETRIES = 3
-PROVIDER = prov.SMK_DEFAULT_PROVIDER
-ENDPOINT = "https://api.smk.dk/api/v1/art/search/"
-LANDING_PAGE_BASE_URL = "https://open.smk.dk/en/artwork/image/"
 IMAGE_SIZE = 2048
```

Comment:
I hadn't thought much about this - everything else is a class attribute on `SmkDataIngester` 🤔 should this be as well?

Reply:
This variable was the one looking more like a constant to me, but I guess to really address #229 it's better to remove it. We don't even need a variable, could make that the param's default value to keep the method **static**.